### PR TITLE
New Docs Landing Page

### PR DESCRIPTION
**Pull request by Alex-Tideman**
## What does this PR do?
Update landing page based on:

https://docs.google.com/document/d/15CjgvKOREuJHjC04IEWmJJpbgDoXZn0NHnbwD29cFsE

https://www.figma.com/file/jTyvfvVy5Ieuhrlqr1bnpN/Web-Redux-2023?node-id=1517%3A21954&mode=dev

live preview: https://temporal-documentation-git-new-landing-page.preview.thundergun.io/

### CODE REVIEW

File: docs/components/Intro.js [hunk 1]
```diff
@@ -1,346 +1,227 @@
...
+    name: "Temporal 101 with Go, Java, TypeScript, Python",
+  },
+  {
+    path: "https://learn.temporal.io/courses/temporal_102/",
+    name: "Temporal 102 with Go",
+  },
+  {
+    path: "https://learn.temporal.io/courses/intro_to_temporal_cloud/",
+    name: "Introduction to Temporal Cloud",
+  },
```
Comment (flossypurse) [inline docs/components/Intro.js:30]: I think we are going to drop these and just have the prereqs, run your first app, and hello world from scratch -

@napcs correct?

File: docs/components/Intro.js [hunk 2]
```diff
@@ -1,346 +1,227 @@
...
-}
+const appDevGuideLinks = [
+  {
+    path: "/dev-guide/go",
+    name: "Go dev guide",
+  },
+  {
+    path: "https://pkg.go.dev/go.temporal.io/sdk",
+    name: "Go SDK API reference",
+  },
```
Comment (flossypurse) [inline docs/components/Intro.js:41]: Let's nix all of the links to API references in this list.
"Start building your first app" should go to dev guides - not api references.
If they click one of these links, the first link on the page is the API ref - so I am not worried about removing them

**Comment by flossypurse** [comment]
suggested reorder to:
- Temporal CLI
- Temporal Web UI
- tctl v1.17

Screenshot 2023-11-02 at 1 52 45 PM

**Comment by flossypurse** [comment]
right now I have to click through the circles to see the stuff 
will it be automated by scrolling?

also... i think the gifs are too much...

Screenshot 2023-11-02 at 1 56 43 PM

File: docs/components/Intro.js (hunk 1 above)
Comment (Alex-Tideman) [reply docs/components/Intro.js]: 👍 dropped these

File: docs/components/Intro.js (hunk 2 above)
Comment (Alex-Tideman) [reply docs/components/Intro.js]: 👍 dropped em

**Comment by Alex-Tideman** [comment]
I'll let you and @canvanooo work out the carousel and the gifs.

**Comment by Alex-Tideman** [comment]
Updated to remove GIFs and new background image

File: docs/components/Intro.js [hunk 3]
```diff
@@ -350,317 +199,220 @@ const displayLinks = (links) => {
...
-            fill="currentColor"
-          />
-        </svg>
-        <h2 className="mb-4 text-xl font-semibold">Operation</h2>
+    <div>
+      <p className="main-subhead">Temporal Platform explanation and usage information.</p>
+      <div className="mb-8">{SdkLogos()}</div>
+      <div className="grid md:grid-cols-2">
+        <SimpleCard title="Get Started with Temporal" content="New to Temporal? Start your journey here by setting up your development environment, running an existing Temporal app, and then building your first app from scratch using our SDKs." links={coreConcepts} />
```
Comment (flossypurse) [inline docs/components/Intro.js:360]:
```suggestion
        <SimpleCard title="Get Started with Temporal" content="New to Temporal? Follow our introductory tutorials and courses to get a feel for the developer experience and the value of the Temporal Platform" links={coreConcepts} />
```

File: docs/components/Intro.js [hunk 4]
```diff
@@ -350,317 +199,220 @@ const displayLinks = (links) => {
...
+        <SimpleCard title="Start building your next app" links={appDevGuideLinks} />
```
Comment (flossypurse) [inline docs/components/Intro.js:361]: Can we add content under this title as well -

> Have a use case? Jump in to the developer guides to start building your next backend application with Temporal .

File: docs/components/Intro.js [hunk 5]
```diff
@@ -1,346 +1,195 @@
...
-      book: true,
-      path: "/namespaces",
-      name: "Namespaces",
-    },
-  ],
-};
+const references = [
+  {
+    path: "/references/events",
+    name: "Events reference",
```
Comment (flossypurse) [inline docs/components/Intro.js:116]:
```suggestion
    name: "Events",
```

File: docs/components/Intro.js [hunk 6]
```diff
@@ -1,346 +1,195 @@
...
+    path: "/references/commands",
+    name: `Commands`,
+  },
+  {
+    path: "/references/sdk-metrics",
+    name: "SDK Metrics",
+  },
+  {
+    path: "/references/configuration",
+    name: `Cluster configuration reference`,
```
Comment (flossypurse) [inline docs/components/Intro.js:128]:
```suggestion
    name: `Self-hosted Cluster config`,
```

File: docs/components/Intro.js [hunk 7]
```diff
@@ -1,346 +1,195 @@
...
+    name: "Set up your development environment",
+  },
+  {
+    path: "https://learn.temporal.io/getting_started/#run-your-first-program",
+    name: "Run your first program",
+  },
+  {
+    path: "https://learn.temporal.io/getting_started/#run-your-first-program",
+    name: "Build an application from scratch",
+  }
```
Comment (flossypurse) [inline docs/components/Intro.js:18]:
```suggestion
  {
    path: "https://learn.temporal.io/getting_started/#run-your-first-program",
    name: "Build an application from scratch",
  },
  {
    path: "https://learn.temporal.io/courses/",
    name: "Enroll in a course",
   }
```

File: docs/components/Intro.js [hunk 8]
```diff
@@ -350,317 +199,220 @@ const displayLinks = (links) => {
...
-                    className="h-7 w-7 pl-1 transition hover:scale-110 code-logo"
-                    src="/img/typescript.svg"
-                    alt="TypeScript logo"
-                  />
-                  <p className="font-normal">TypeScript</p>
-                </div>
-              </Link>
-            </li>
-          </ul>
+          <h1>Get started with Cloud</h1>
```
Comment (flossypurse) [inline docs/components/Intro.js:238]:
```suggestion
          <h1>Is Temporal Cloud right for you?</h1>
```

I think we need to update this header to match the call to action.

**Review by flossypurse — APPROVED** [review]
Looking good now.
Checking on timing for publishing.

File: docs/components/Intro.js [hunk 9]
```diff
@@ -350,317 +186,319 @@ const displayLinks = (links) => {
...
+      <div className="mb-8">{SdkLogos()}</div>
+      <div className="grid md:grid-cols-2">
+        <SimpleCard
+          title="Get started with Temporal"
+          content="New to Temporal? Follow our introductory tutorials and courses to get a feel for the developer experience and the value of the Temporal Platform."
+          links={coreConcepts}
+        />
+        <SimpleCard
+          title="Start building your next app"
+          content="Have a use case? Jump in to the developer guides to start building your next backend application with Temporal."
```
Comment (djmagee) [inline docs/components/Intro.js:412]:
```suggestion
          content="Have a use case? Jump into the developer guides to start building your next backend application with Temporal."
```

File: docs/components/Intro.js [hunk 10]
```diff
@@ -350,317 +186,319 @@ const displayLinks = (links) => {
...
+      <CloudCard links={cloudInfo} />
+      <div className="grid md:grid-cols-2">
+        <SimpleCard
+          title="Production readiness"
+          content="Temporal Cluster deployment information and how-to guides."
+          links={clusterGuide}
+        />
+        <SimpleCard
+          title="Dev tools"
+          content="Use Temporal’s CLI or Web UI to manage and monitor your Workflows."
```
Comment (djmagee) [inline docs/components/Intro.js:425]:
```suggestion
          content="Use the Temporal CLI or Web UI to manage and monitor your Workflows."
```

**Comment by flossypurse** [comment]
commits merged in this PR: https://github.com/temporalio/documentation/pull/2463

**Pull request closed without merging**